armada/tests/unit/utils/test_git.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 The Armada Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import os
+
+from armada.utils import git
+
+class TestGitUtils(unittest.TestCase):
+    def test_git_clone_repo_pass(self):
+        url = 'http://github.com/att-comdev/armada'
+        dir = git.git_clone(url)
```

Comment:
definitely works but might be better to mock the functions that actually download a repo to avoid downloading/deleting on every run of the test